cirq/ops/parity_gates.py

```diff
@@ -39,6 +39,16 @@ class XXPowGate(eigen_gate.EigenGate,
               [0 1 0 0]
               [1 0 0 0]
 
+    The XX**t gate implements the following unitary:
```

Comment:
To be consistent with existing documentation, let's do it in this manner: https://github.com/quantumlib/Cirq/blob/144e1ebdd6196cf6c8ea8e6f497dbb947bf57142/cirq/ops/common_gates.py#L56

Reply:
I don't think this got resolved.  Was your latest commit uploaded?